pkg/sqlite/database.go

```diff
@@ -202,9 +192,9 @@ func (db *Database) open(disableForeignKeys bool) (*sqlx.DB, error) {
 	}
 
 	conn, err := sqlx.Open(sqlite3Driver, url)
-	conn.SetMaxOpenConns(25)
-	conn.SetMaxIdleConns(4)
-	conn.SetConnMaxLifetime(30 * time.Second)
+	conn.SetMaxOpenConns(10)
+	conn.SetMaxIdleConns(10)
+	conn.SetConnMaxIdleTime(30 * time.Second)
```

Comment:
Since there's now reasons for choosing these particular values, could you make them constants with an explanation for why each value was chosen?

Reply:
Added both constants